docs/schemas/arrays.md

```diff
@@ -487,16 +553,16 @@ Since arrays are unlike other data types in that they usually require some speci
 and many formats don't have a single canonical array type, generators may accommodate multiple array
 representations.
 
-The pydantic generator currently supports a "List of lists" style array representation to be able to use
+The basic representation supported by the pydantic generator is "List of lists" style array representation to be able to use
 arrays without any additional external dependencies beyond pydantic. 
```

Comment:
```suggestion
The basic representation supported by the pydantic generator is the "List of lists" style array representation. This can be used without any additional dependencies beyond pydantic.
```